### PR TITLE
9998 CAV/Submitted Cases Feedback

### PR DESCRIPTION
**Pull request by ujahio**
Addressed feedback on populating the `CAV` and `Submitted Cases`.

Solution:  Move pagination calculation of table for CAV and Submitted cases to `getCasesByStatusAndByJudgeInteractor`. The number of `CAV` and `Submitted Cases`, after removing cases with unwanted docket entries, should be used to evaluate cases to paginate through.

### CODE REVIEW

File: shared/src/business/entities/judgeActivityReport/JudgeActivityReportSearch.ts [hunk 1]
```diff
@@ -83,12 +85,18 @@ export class JudgeActivityReportSearch extends JoiValidationEntity {
...
       .number()
       .optional()
       .description(
         'The page size for getting aggregated judge activity report for closed cases, opinions, orders issued  and sessions held',
       ),
+    pageSize: joi
+      .number()
+      .optional()
+      .description(
+        'The page number for calculating the number of cav and submitted cases to display',
```
Comment (TomElliottFlexion) [inline shared/src/business/entities/judgeActivityReport/JudgeActivityReportSearch.ts:98]: Nit

**Review by TomElliottFlexion — APPROVED** [review]
Might it be worth checking this in a integration test?

**Review by jasonfu9966 — APPROVED** [review]

File: shared/src/business/entities/judgeActivityReport/JudgeActivityReportSearch.ts (hunk 1 above)
Comment (ujahio) [reply shared/src/business/entities/judgeActivityReport/JudgeActivityReportSearch.ts:98]: Will be fixed on the integration test branch being worked on now.

**Comment by ujahio** [comment]
> Might it be worth checking this in a integration test?

Agreed. Working on the integration test in a different branch.

**Pull request merged**